core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -124,6 +124,19 @@ <h3>Update username</h3>
     </form>
   </div>
 </md-card>
+
+<md-card class="oppia-admin-page-card oppia-page-card oppia-long-text">
+  <h3>Number of pending deletion requests</h3>
+  <div class="oppia-query-form">
+    <form class="form"
+          ng-submit="$ctrl.getNumberOfPendingDeletionRequestModels()">
+      <div class="form-group row">
+        <input type="submit" value="Get number of users that are being deleted">
```

Comment:
that are being deleted --> whose deletion is currently in-progress

(I think)

Reply:
Done.